app/src/main/res/layout/fragment_send_new.xml

```diff
@@ -16,6 +16,7 @@
         app:layout_constraintTop_toTopOf="parent"
         app:navigationIcon="@drawable/ic_back_night"
         app:titleTextAppearance="@style/UiKit.TextAppearance.SemiBold.Title3"
+        app:titleTextColor="@color/text_night"
```

Comment:
Can you make a minor research, why the texts are not #2B2B2B by default? 
The text color is set inside `UiKit.TextAppearance`

Reply:
we had an agreement that the color is affixed separately as it changes and it was not recorded in the style

Reply:
i can confirm that
we can make it default and override it

Reply:
ok let's do that
 i will

Reply:
heh lol
it is already there
but
here we have 
app:titleTextColor="@color/text_night"

Reply:
and here is just 
![image](https://user-images.githubusercontent.com/99185280/212646731-2f0e0504-4dc0-4dd2-99ff-56c5e8638e3e.png)
       <item name="android:textColor">@color/text_night</item>

Reply:
@1sakov @gslevinkov ^^

Reply:
@kabramovp2p textcolor is still not working in textviews.

Reply:
@gslevinkov let's make the default color `text_night`
If there is a different text, we will override in xml directly

Reply:
separate task or..?

Reply:
yes separate. 
@kabramovp2p you can take it if you want

Reply:
https://p2pvalidator.atlassian.net/browse/PWN-6848